build.sh

```diff
@@ -32,7 +33,7 @@ else
   exit 1
 fi
 
-LANGUAGE="python"
+RAY_BUILD_PYTHON="YES"
```

Comment:
It's better to also initialize `RAY_BUILD_JAVA` here, otherwise it will be an empty string if `-l` isn't specified.